This has past the CI build but locally I am seeing a failure,

```
$ rm -r build && ant build test
...
test.sbt:
[stopwatch] [quick.sbt-interface.timer: 0.229 sec]

bc.init:

bc.prepare:

BUILD FAILED
/home/jdb/workspaces/public/scala/build.xml:1545: java.lang.ClassCastException: org.codehaus.plexus.DefaultPlexusContainer cannot be cast to org.codehaus.plexus.PlexusContainer
```

--- a/build.xml
+++ b/build.xml
@@ -461,7 +461,7 @@ TODO:
     </fail>
 
     <!-- Allow this to be overridden simply -->
-    <property name="sbt.latest.version"    value="0.12.4"/>
+    <property name="sbt.latest.version"    value="0.13.9"/>
 
     <property name="sbt.src.dir"           value="${build-sbt.dir}/${sbt.latest.version}/src"/>
     <property name="sbt.lib.dir"           value="${build-sbt.dir}/${sbt.latest.version}/lib"/>

